### PR TITLE
Fix type annotatons

### PR DESCRIPTION
**Pull request by mxr**
This fixes a few lint errors I saw in pyupgrade (only flagged by IntelliJ)

image

image

~https://user-images.githubusercontent.com/252209/60769913-666c5580-a0a3-11e9-98bd-a392743fd381.png~ will follow up about that one separately

### CODE REVIEW

File: tokenize_rt.py [hunk 1]
```diff
@@ -26,7 +29,7 @@ class Offset(collections.namedtuple('Offset', ('line', 'utf8_byte_offset'))):
     __slots__ = ()
 
     def __new__(cls, line=None, utf8_byte_offset=None):
-        # type: (Optional[int], Optional[int]) -> None
+        # type: (Type['Offset'], Optional[int], Optional[int]) -> 'Offset'
```
Comment (asottile) [inline tokenize_rt.py]: do you need the first argument? it should be inferred for you

Comment (mxr) [reply tokenize_rt.py]: IntelliJ gives the wrong hint otherwise (3rd screenshot). I can remove it and just followup with IntelliJ

Comment (asottile) [reply tokenize_rt.py]: ah yeah, pycharm has their own type checker that probably doesn't understand `__new__` (if I had to guess)

Comment (mxr) [reply tokenize_rt.py]: OK removed it. PR should be green now 🤞

**Pull request merged**